WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -1410,17 +1391,17 @@ title_lawspeaker = {
 		OR = {
 			religion = norse_pagan
 			religion = norse_pagan_reformed
+			religion = germanic_pagan
+			religion = germanic_pagan_reformed
 		}
 	}
 	
 	allowed_to_grant = {
 		OR = {
-			is_feudal = yes
-			is_tribal = yes
-		}
-		OR = {
```

Comment:
@loup99 

I removed the "OR" under Feudal and Tribal because that would then allow any government to use that title. By removing that entire line, I removed government restrictions.